backend/package.json

```diff
@@ -20,10 +19,9 @@
         "express": "^4.18.2",
         "express-session": "^1.17.3",
         "express-validator": "^7.0.1",
-        "jsonwebtoken": "^9.0.2",
-        "mongoose": "^7.5.3"
+        "jsonwebtoken": "^9.0.2"
     },
-    "proxy": "http://localhost:5000",
+    "proxy": "http://108.141.158.1:5000",
```

Comment:
Is this correct? The message you send me stated: `The "proxy" field in your package.json file is used to specify a backend server for the development environment.`